sdk/python/arvados/keep.py

```diff
@@ -375,6 +375,8 @@ def get(self, locator, method="GET", timeout=None):
                     curl.setopt(pycurl.HEADERFUNCTION, self._headerfunction)
                     if self.insecure:
                         curl.setopt(pycurl.SSL_VERIFYPEER, 0)
+                    else:
+                        curl.setopt(pycurl.CAPATH,os.path.dirname(arvados.util.ca_certs_path()))
```

Comment:
Shouldn't this be CAINFO?

https://curl.haxx.se/libcurl/c/CURLOPT_CAPATH.html
"string naming a directory holding multiple CA certificates to verify the peer with."

https://curl.haxx.se/libcurl/c/CURLOPT_CAINFO.html
"string naming a file holding one or more certificates to verify the peer with."